projects/batfish/src/org/batfish/grammar/cisco/CiscoLexer.g4

```diff
@@ -2391,6 +2412,11 @@ GROUP_URL
    'group-url'
 ;
 
+GSHUT
```

Comment:
Well I'll be damned. I was thrown off by the fact that it's the only all-caps community I'm aware of. Sorry about that.